network.go

```diff
@@ -79,6 +92,15 @@ func GetIP(r *http.Request, options ...Options) net.IP {
 	return net.ParseIP(host)
 }
 
+// GetJWTSub returns sub from request JWT.
+func GetJWTSub(r *http.Request, secret string) (string, error) {
+	if token, valid := getAuthorizationToken(r); valid {
```

Comment:
Please reverse your condition handling to have the edge/corner case of the right.

```suggestion
	token, valid := getAuthorizationToken(r)
	if !valid {
		return "", ErrInvalidJWT
```